app.py

```diff
@@ -0,0 +1,11 @@
+
+from flask import Flask, request, jsonify
+#from flask_marshmallow import Marshmallow
```

Comment:
you just commented out the import, commented code is confusing in a code base, better to just remove it. If someone reads your code, they will waste time wondering why the import and why is it commented, so we should just remove it